### PR TITLE
Add samples and consumers

### PR DESCRIPTION
**Pull request by tharindulak**
## Goals
To give an understanding on how to use the product.  

## Approach
Added new samples with the guides.

### CODE REVIEW

File: modules/samples/artifacts/HBaseSample/HbaseSample.siddhi [hunk 1]
```diff
@@ -0,0 +1,100 @@
+@App:name("HbaseSample")
```
Comment (ramindu90) [inline modules/samples/artifacts/HBaseSample/HbaseSample.siddhi]: Can we change app name to store-hbase.siddhi

File: modules/samples/artifacts/MongoSample/MongoSample.siddhi [hunk 2]
```diff
@@ -0,0 +1,96 @@
+@App:name("MongoSample")
```
Comment (ramindu90) [inline modules/samples/artifacts/MongoSample/MongoSample.siddhi]: Can we change app name to store-mongodb.siddhi and change the folder name to MongoDBSample

File: modules/samples/artifacts/RDBMSSample/RdbmsSample.siddhi [hunk 3]
```diff
@@ -0,0 +1,100 @@
+@App:name("RdbmsSample")
```
Comment (ramindu90) [inline modules/samples/artifacts/RDBMSSample/RdbmsSample.siddhi]: Can we change app name to store-rdbms.siddhi

File: modules/samples/artifacts/ScriptJsSample/Script-js-sample.siddhi [hunk 4]
```diff
@@ -0,0 +1,42 @@
+@App:name('Script-js-sample')
```
Comment (ramindu90) [inline modules/samples/artifacts/ScriptJsSample/Script-js-sample.siddhi]: Can we change app name to script-js.siddhi

File: modules/samples/artifacts/SolrSample/SolrSample.siddhi [hunk 5]
```diff
@@ -0,0 +1,93 @@
+@App:name("SolrSample")
```
Comment (ramindu90) [inline modules/samples/artifacts/SolrSample/SolrSample.siddhi]: Can we change app name to store-solr.siddhi

Comment (tharindulak) [reply modules/samples/artifacts/SolrSample/SolrSample.siddhi]: noted

File: modules/samples/artifacts/JmsSample/jms-sample.siddhi [hunk 6]
```diff
@@ -0,0 +1,71 @@
+@App:name('jms-sample')
```
Comment (ramindu90) [inline modules/samples/artifacts/JmsSample/jms-sample.siddhi]: this sample name should be aligned with other transports eg: ReceiveHTTPInJsonFormatWithCustomMapping

File: modules/samples/artifacts/ScriptJsSample/Script-js-sample.siddhi [hunk 7]
```diff
@@ -0,0 +1,42 @@
+@App:name('Strocript-js-sample')
```
Comment (ramindu90) [inline modules/samples/artifacts/ScriptJsSample/Script-js-sample.siddhi]: this file is duplicated

File: modules/samples/sample-clients/jms-producer/pom.xml [hunk 8]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ramindu90) [inline modules/samples/sample-clients/jms-producer/pom.xml]: update licenses header to current

File: modules/samples/sample-clients/jms-producer/src/main/java/org/wso2/sp/sample/jms/client/JmsClient.java [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+            Thread.sleep(2000);
+            siddhiAppRuntime.shutdown();
+            Thread.sleep(2000);
+        }
+    }
+
+    private static List<String[]> readFile(String fileName) throws IOException {
+        File file = new File(fileName);
+        Scanner inputStream = new Scanner(file);
+
```
Comment (ramindu90) [inline modules/samples/sample-clients/jms-producer/src/main/java/org/wso2/sp/sample/jms/client/JmsClient.java]: remove unnecessary spaces

**Comment by appellj** [comment]
I don't understand this description; please clarify:

+    contains - containsSweetProductionStream (to check whether a particular data contains in the store)

**Comment by ramindu90** [comment]
@appellj : when sample siddhi app uses "containsSweetProductionStream" it will be used in the query which checks the attribute values in containsSweetProductionStream exists in the SweetProductionTable

**Comment by appellj** [comment]
Thanks! I'll revise the description to: "verifies whether an attribute in the stream exists in the store"

**Pull request merged**